.travis.yml

```diff
@@ -1,21 +1,17 @@
 language: php
```

Comment:
Would you accept a PR using GithubActions instead of Travis?

Reply:
I find it difficult and sometimes troublesome to work with Travis, so yeah would definitelly consider it. Thanks!

Reply:
I will create a PR after my vacation 👍🏻